react-training/src/App.js

```diff
@@ -0,0 +1,16 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
I can't see bootstrap module in the src/ folder or in the package.json
The project is not compiling on my local